archetypes/custom-module/src/main/resources/archetype-resources/src/main/java/Attribute.java

```diff
@@ -0,0 +1,39 @@
+package ${package};
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum Attribute {
+   // must be first
+   UNKNOWN(null),
+
+   MESSAGE("message");
+   private static final Map<String, Attribute> ATTRIBUTES;
+
+   static {
+      final Map<String, Attribute> map = new HashMap<>(64);
```

Comment:
remove the size from here and use `Map.copyOf()` when assigning to the `ATTRIBUTES` field